feature/history/src/main/java/com/yapp/feature/history/components/SessionAttendanceItem.kt

```
@@ -0,0 +1,11 @@
+package com.yapp.feature.history.components
+
+import androidx.compose.runtime.Composable
+
+@Composable
+internal fun SessionAttendanceItem(
+    date: String,
+    time: String? = null,
+) {
+
+}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)